pkgs/by-name/sk/skimpdf/package.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, undmg
+, fetchurl
+,
+}:
+stdenv.mkDerivation rec {
+  pname = "Skim-${version}";
+  version = "1.6.21";
+
+  src = fetchurl {
+    name = "Skim-${version}.dmg";
+    url = "mirror://sourceforge/project/skim-app/Skim/Skim-${version}/Skim-${version}.dmg";
+    sha256 = "0SeS41ExCZfGmjiK2KvvdFpP7IVzm7xHKCaKmEUYQAY=";
+  };
+
+  nativeBuildInputs = [ undmg ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/Applications
+    cp -R *.app $out/Applications
```

```suggestion
    runHook preInstall
    mkdir -p $out/Applications
    cp -R *.app $out/Applications
    runHook postInstall
```
https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases